3scale/3scale-operator
# Removed reference to the masterCredentialsRef in the tenant CRD and use the APIManager namespace instead

# What
Removed reference to the masterCredentialsRef in the tenant CRD and use the APIManager namespace instead

# Verification
- deploy this branch locally in the a `3scale-test` namespace
- Create an apimanager
- create a secret in the apimanager namespace
![image](https://user-images.githubusercontent.com/16667688/162991311-9ec8b693-c2c6-4998-89eb-dd374cb660d6.png)
- create a tenant CR with the follow spec
```yaml
spec:
  apiManagerNamespaceRef: 3scale-test
  email: tenant1@emails.com
  organizationName: tenant1-org
  passwordCredentialsRef:
    name: tenant-5-admin-password
    namespace: 3scale-test
  systemMasterUrl: '<master url e.g. https://master.apps.aucunnin.egcw.s1.devshift.org/'>
  tenantSecretRef:
    name: provider-account-ref
    namespace: 3scale-test
  username: tenant1
```
- After the tenant reconciles you should see the status update e.g.
```yaml
status:
  adminId: 5
  tenantId: 4
```
